app/views/docs/permissions.phtml

```diff
@@ -44,60 +44,62 @@
     </thead>
     <tbody>
     <tr>
-        <td>
-            role:all
-            <p class="margin-top-small text-fade">version >= 0.12</p>
-        </td>
+        <td>Role.any()</td>
         <td>Wildcard permission. Gives anyone read or write access.</td>
     </tr>
     <tr>
-        <td>user:[USER_ID]</td>
+        <td>Role.guests()</td>
+        <td>Access to any guest user with an anonymous session. Logged in users don't have access to this role.</td>
+    </tr>
+    <tr>
+        <td>Role.users()</td>
+        <td>Access to authenticated users.</td>
```

Comment:
If an above comment is true, then here we should mention that anonymous sessions also count as users.

Reply:
Yes, afaik, we just check for a. session vs no session in code, so anonymous are still users.